contracts/fungible-token-full-interface.aes

```diff
@@ -1,4 +1,4 @@
-@compiler >= 4
+@compiler >= 5
```

Comment:
that shouldn't be needed for the interface

Reply:
resolved: https://github.com/mradkov/aeternity-fungible-token/pull/64/commits/47289b4882e807d2f29f4bd7441ef285b2302fc1